app/index.jsx

```diff
@@ -10,13 +10,17 @@ import { // eslint-disable-line import/no-extraneous-dependencies,import/no-unre
     DrawTools,
     ChartSetting,
     Share,
+    ChartTitle,
+    AssetInformation,
+    ComparisonList,
 } from '@binary-com/smartcharts'; // eslint-disable-line import/no-unresolved
 import React from 'react';
 import ReactDOM from 'react-dom';
 import { configure } from 'mobx';
 import './app.scss';
 import './doorbell';
 import { ConnectionManager, StreamManager } from './connection';
+import Notification from './libs/Notification.jsx';
```

Comment:
just put it as part of `app`; why do we need a `lib` folder for 1 file?

Reply:
@brucebinary  sure, but I have a problem with passing messaging between chart and notification component, I am thinking about using mobx here.